.github/workflows/test-macos.yml

```diff
@@ -0,0 +1,20 @@
+name: CI Build MacOS
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+      - name: Set up Shellcheck
+        run: brew install shellcheck
+      - name: Build project with Maven
+        run: mvn -B -ntp install
```

Comment:
For the record: thanks for copying / using the flags for sustainability (`-B -ntp`) - see https://issues.apache.org/jira/browse/MNG-7338